cookbook-react/recipes/inputs/barcelona/index.test.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import TestRenderer from 'react-test-renderer';
+import { render, queryByAttribute }  from '@testing-library/react';
+
+import Input from '.';
+
+const getById = queryByAttribute.bind(null, 'data-testid');
```

Comment:
Can you use `https://testing-library.com/docs/dom-testing-library/api-queries#bytestid`?

But what do you think of using `getByRole` instead of adding test ids?
Based on `testing-library` recommendations https://testing-library.com/docs/guide-which-query